### ISSUE TITLE
.

### ISSUE THREAD

**Issue by ghost** [n00mkrad/flowframes#148]
.

**Comment by n00mkrad** [comment]
Not a bug, that's necessary to know that it's FPS-limited

**Comment by n00mkrad** [comment]
Or does this also happen when the output framerate is lower than the maximum?

**Comment by curya** [comment]
Right. It only happens when the output framerate is lower, e.g. I take a 30fps video and make it 120fps, but limit framerate to 60, then the output file will have "-fpsLimit". If I don't limit the framerate, no fps shows in the filename at all (as long as you remove "[FPS]fps" in settings).

**Comment by n00mkrad** [comment]
Yeah, exactly, because you removed the FPS variable, it will only append "fpsLimit".

If it wouldn't do that, the option "Create Outputs With Full And Limited FPS" would overwrite the first file.

**Comment by curya** [comment]
I see. Is there any way we could get a feature to remove "-fpsLimit" since we can already remove "[FPS]fps"?

**Comment by n00mkrad** [comment]
OP deleted all info.
